apps/fortnitestore/fortnite_store.star

```diff
@@ -0,0 +1,109 @@
WKESRWf6pZ+ARBOGTAB7nrYuDCwAcHBzLLgLYn8+eOPFyCPgFQaUgkIF4KjT+5qRjix89zwa+6ZMZnJrOQFqbRdwi8d3vWov0OR29d7VZe1NRbUj3E/SZsWwDeQBQA1F+6S7FepfxEcjIZguYyoVwIlfGVE5FVlUxfNJdfyAbB0Cw5Sa2n65ExCc1yBGANrN96k1RyBEZFCYoTIiRCORYHEo0iuYVCmKyjFJJhxwWnr1stN8SZwaPv/BiPK5/nhCK6dkJfHf46/hwBJDLcXznlT9Bah3LEaAKrgU6c2ICwyf2YsstLOHSYw//EbqHfggA2NTZh2QiDcgqCrkc1FJty3V39050d+/Eu7t2YO/eb6D/ZL+fN0VkJ/lfsApANBpFZCW73qOvD2HrXb1VFhTI7HexWUE8FYOyQkTzqjjy56zzqVijiMjiQfQ5FjPyoXsfwaM7GfnPZt02FAk0uLLmT8IXMvwTsojsErvDJyYxOT0G0ixBWElBminEcOMRTQxTGDpAK5agJcmOF4tmGNDmvdsaQNB3hOj+lJJ2TLt1mk4d64r1RSBov1ouC5a88v9fEgyhD6wfnK5SK+beSZwVjMEPLS0JwEIxvFVT7YV2X2t+vdiEoXZNblGoeQaEiJBkxSMCiKBhAgpAkESQpgjEJhmyvBKyHIUsx5GSWWeYEFd8kY8sHNcrNm7cWARwsDxz/DGYQGFgD9IyoCWjoN44/LJ/xJfIArEATi4iz9Pqtbpbn+0+2O7fKLHIP2Ux/nbG/xCh0CoUJATouuGQfTsHgL2sWeOWZo91RqBygJ1sNODy78sJUKt/DnRgui7ZHyCIFGphBlop9l7eoji4AMDBwXFpxQDgWPC7zne1g4kAjPgPHmQksHVd2iWK71qLR+/rgTrjryS4f3Cq4fE6o0vLqH0qW6giFG1JGW1JRvzyhbwjAAydBFLzbOAe+gkbWM9QETcrBtqUi8gsHMLE5b8zlaP2p9ZEBxMBBr7uI79BxFYlMDzZD+xwv+vrfdT5nJvNwNDUhkednGTxmu3tPQAeR//f9F8V7fRxy+2/0f9fKk5NT111zyMJU9AKgRA2XBHgYmBn/zeu8wI9ocYiAWmSfdn/65Uyq4O3PrHmln/gownH9QqtcPouhEJ/Rc8X7yqNu15VRJJAJAlU0/xCwIXCS/4tkdIrwNYcvz2u/ppF8HXdUxYw4AVgk387gaBmXJ44KcM6p1AolOYtioMLABwcHJcUzefO7Z1buXJJ29jk38bAK4xIHXzlBBYaKieiim9ZDszLlRWBrL2hwERc8HsUNEtA9zrrvNYBM3PMktz9Ex3DPxExVJQxVSJoUwRsS7J9k+IsaNSNV18gD8CZrSubLns8Xs/67ZPfHX5iEkD7YreJxxLA2zqGJ/vR3d7nkP4LweTkINrbe/D4h/4I3/jeby0qBECWl+4/39a6FidOTTe+Fg2OOzpxyPm8qWMrIkozyqXF57UkzdSx/l+VIsDFFKAMG67pzBECeJ+3VIhNMiTL+h9NpNC8ugXRRMr5/U4l8U1+lTiuc/J/SJubhTrJcrKI9vNRyoLqOohkWbotck0vNvmIYQCUVuUQaqwd+Mm/VsPVP0j+vdApBQldWHlivU6F9Do5AUq8VXFwAYCDg2NZcTJ7+r43Z7I7T/7kDNalbuyeU5hp3GhOwmhO4plvf91Zd/zNSeQq1fuwyf/wm6ew58kDjgAAAGql8fGVqD8JQEuA39+cDOYM8K9PA+vrAYKUspKyb7lJw5abNHRlJBzIiZgqERzIAdvezeLoxelxCHNngGhbw/M1gR9cubtlPgvg15a61dAxVwBYCIODB3BqegprW9vQ07OtLvlebCLArXf14tAry5sDoJ4AMDw8hD0DT/gEgA9u/wXEkyxJXyMhoJ71305ASCuX6RYL1gQwUN7OFgEuivxfFzOORVyjUCDzf/CeW9Z/0mRl416a9R8GhH/hIwvHdUr+P4VQ6K+0uVnoxdnafaoouiKAqoJSykqgXqgIYJF/33ksYP0PHssm/zVzANiWf00DNYlD/i817HMIhUKjvGVxcAGAg4NjWbG2ZeW317ZsvOml4mlgLofXzrnW19TGzWi/owdPPvW0PXSjpFPk5xirjhWBR9+9BUjFMNg/jMH+ITz1gxwAGdE1bPIsrPYT9ojfYI8zZb8L+qmAR/qh14ImzzLkNe4EPtU0i9QqdzkW9ludg1UI0uk80mlg9qyO8akSiqUUplt3AbEuRIvj2GrMAqoKVYlD1YBYPGYSWUY0noSqqpivhH58Oe+PNz/D3qH/8SyUxK/lSzOOdT2eSqJcLkFVS9YdCkwiZAAicQoqkoAxQfIUWjx0vB/PjP45soUpIAOcKn0MH3vwc87vhZkMZDmOx3b9Ib7+nf+IgdefQ99PPYxSIQ9dZTfOmwwpIsvIzeaRPZPHbD6PQiEPuTnqzzdg3aCoJKMkx5FKxbAqqiAyyxqKqgLySoDqgK4DXTfeis3vuhdqjfIS/T/+Lo6cPghpBftPR/7tIIpkFo8/9BV2PtIKUGpA9+QhkCPWdZEEqFAhCBokCRDl2kJA0PtTm7e+sHikau+6YoA0qlO92Emg6bmjBFiy474QiDkN7MAkVew10ADt0AN7w1C1JNbw/C/6CbjIC0jcXdh/wS77FSLu/3XKA9LAfQ+QfxEgEQmisoL9nowhEo0iqjBPplubNuz9eDL5449buTu8zy8Hx7WM+bNnviwa+hf0s7PA3BmgOGuVsrOZtAaiApTqIIZl+ScEhBAAGgCjKmcJaDCnSaBHDVjmqSUkGJ717W0Mw/1sk2taZrZ4Yn2vqhbZpxS6YYDOG462QE3irKdbI61mOgdixwh2VyFa1Z0xh4PaI0PZdP8PUWKQ5BaY4VU/4K2LgwsAHBwcy4Y/ffIHm/7wD75+07Zt9wAADhw4iE6PCX5mYgSp1FpsvmMTnnzqaWy+fRPSrYz8Z07lkW6NoaePFVwf7B/CYP8womtYHfbmFkb
```

Comment:
This isn't a huge issue, but if you scale that graphic down a bit before base64 encoding we'll get a bit less data in the .star file which is nice when reviewing. =)